doc/start/hardware-recommendations.rst

```diff
@@ -21,21 +21,28 @@ data cluster (e.g., OpenStack, CloudStack, etc).
 CPU
 ===
 
-CephFS metadata servers are CPU intensive, so they should have significant
-processing power (e.g., quad core or better CPUs) and benefit from higher clock
-rate (frequency in GHz). Ceph OSDs run the :term:`RADOS` service, calculate
-data placement with :term:`CRUSH`, replicate data, and maintain their own copy of the
-cluster map. Therefore, OSD nodes should have a reasonable amount of processing
-power. Requirements vary by use-case; a starting point might be one core per
-OSD for light / archival usage, and two cores per OSD for heavy workloads such
-as RBD volumes attached to VMs.  Monitor / manager nodes do not have heavy CPU
-demands so a modest processor can be chosen for them.  Also consider whether the
-host machine will run CPU-intensive processes in addition to Ceph daemons. For
-example, if your hosts will run computing VMs (e.g., OpenStack Nova), you will
-need to ensure that these other processes leave sufficient processing power for
-Ceph daemons. We recommend running additional CPU-intensive processes on
-separate hosts to avoid resource contention.
-
+CephFS metadata servers are CPU intensive. CephFS metadata servers should
+therefore have quad-core or better CPUs and high clock rates (GHz). OSD nodes
+need hardware with enough processing power to run the RADOS service, calculate
+data placement with CRUSH, replicate data, and maintain their own copies of the
+cluster map.
+
+The requirements of one Ceph cluster are not the same as the requirements of
+another, but here are some general guidelines. Around 2014, we recommended 
+that  you have 2 cores per OSD in your Ceph cluster, but this is no longer
```

Comment:
FWIW, in 2014 I commonly heard 1 core/OSD.  HT wasn't usually explicitly mentioned, but was usually encouraged, so that was effectively 1 *thread* per OSD.

Reply:
@markhpc, I'm going to defer to you on this one. What do you say?

Reply:
Sorry for the delay, agreed 100%.  2 cores per OSD is easily enough for HDDs, but for flash storage it's just one point (that's fairly low) on the performance curve.  Certainly enough to make the OSD function (though you can probably function with 1 core as well), but nowhere near where we top out if pursuing performance with fast devices.

Reply:
Agreed.